System/RaspberryPi/GPIO.hs

```diff
@@ -371,3 +393,22 @@ setClockDividerAUXSPI a = c_setClockDividerAUXSPI $ fromIntegral a
 -- on MOSI, and simultaneously clocks in data from MISO. Returns the read data byte from the slave.
 transferAUXSPI :: Word8 -> IO Word8
 transferAUXSPI input = fromIntegral <$> c_transferAUXSPI (fromIntegral input)
+
+-------------------------------------------- PWM functions -------------------------------------------------------------------------
+
+-- |Sets the PWM clock divisor, to control the basic PWM pulse widths. 
+setClockPWM :: Word32 -> IO ()
+setClockPWM a = c_setClockPWM $ fromIntegral a
+
+-- |Sets the PWM pulse ratio to emit to DATA/RANGE, where RANGE is set by 'setRangePWM'.

```

Comment:
This comment does not match the name of the function but looks like it belongs to the `setRangePWM` function?